apis/validator/block.v3.yaml

```diff
@@ -43,6 +43,39 @@ get:
         is a flag and does not take a value.
       schema: {}
       allowEmptyValue: true
+    - name: builder_comparison_factor
+      in: query
+      required: false
+      description: |
+        Percentage multiplier to apply to the builder's payload value when choosing between a
+        builder payload header and a local payload. This parameter is only relevant if the beacon
+        node is connected to a builder, deems it safe to produce a builder payload, and receives
+        valid responses from both the builder endpoint _and_ the local execution node. When these
+        preconditions are met, the server MUST act as follows:
+
+        * if `local_payload_value >= builder_comparison_factor * builder_payload_value // 100`, then
+          return a full (unblinded) block containing the local payload.
```

Comment:
`local` gets used in many ways -- my understanding is that the precise distinction is builder API or engine API?

It might not be local as such, even for engine API.

Reply:
good point, clarified in e5b1045ecf5ed474a38ac857937dc593efee269c